src/main/java/org/checkerframework/specimin/UnsolvedSymbolVisitor.java

```diff
@@ -58,6 +58,7 @@
 import java.util.Map;
 import java.util.Optional;
 import java.util.Set;
+import javax.swing.*;
```

Comment:
Two notes about this:
* avoid wildcard imports
* I don't see why you are importing anything from `javax.swing`, which is a GUI library